README.md

```diff
@@ -61,7 +61,7 @@ Clone the repository:
 ```
 git clone git@github.com:googlesamples/android-architecture.git
 ```
-This also checks out the master branch. If you want to change to a different sample: 
+This step checks out the master branch. If you want to change to a different sample: 
```

Comment:
I would probalby re-phrase the previous text to:
Clone the repository and check out the master branch:

If you want to change to a different branch: